maven/impl-projects/src/main/java/org/jboss/forge/addon/maven/projects/archetype/CustomMavenArchetypeProjectType.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
The license header is wrong. It should be: 

    /**
     * Copyright 2016 Red Hat, Inc. and/or its affiliates.
     *
     * Licensed under the Eclipse Public License version 1.0, available at
     * http://www.eclipse.org/legal/epl-v10.html
     */